static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -210,7 +212,7 @@ const TabDivider = styled('div')`
   width: 1px;
   border-radius: 6px;
   background-color: ${p => p.theme.gray200};
-  margin: 9px auto;
+  margin: 8px 4px;
```

Comment:
This is applied to the tab divider element - it ensures that there's a little bit of gap between the shadow of a tab and the divider:
<img width="166" alt="image" src="https://github.com/user-attachments/assets/fd76e489-8faa-41ef-ad04-a621b59da5b4">

Without the margin, it looks like this:

<img width="149" alt="image" src="https://github.com/user-attachments/assets/cbd3b3b1-fc1b-4e44-a0a6-ed6063b430dd">